docs/guides/email/02-basic-email-system.md

```diff
@@ -448,34 +479,30 @@ query = SELECT email FROM virtual_users WHERE email='%s'
 
 !!! warning 
 
-    If you encounter this kind of error after running `systemctl start postfix.service` -- "fatal: open lock file /var/lib/postfix/master.lock: unable to set exclusive lock: Resource temporarily unavailable." Please delete the existing **/var/lib/postfix/master.lock** file
+    If you encounter this kind of error after running `systemctl start postfix.service`: `fatal: open lock file /var/lib/postfix/master.lock: unable to set exclusive lock: Resource temporarily unavailable.` Please delete the existing `/var/lib/postfix/master.lock` file
 
 Testing Postfix configure:
 
 ```bash
-Shell(192.168.100.6) > systemctl start postfix.service
-Shell(192.168.100.6) > postfix check
-Shell(192.168.100.6) > postfix status
+Shell(192.168.100.7) > systemctl start postfix.service
```

Comment:
@jimcat8  :

-  what is the correct IP that should appear here -  192.168.100.6 (original) OR 192.168.100.7 (new) ?